packages/react-devtools-shared/src/backend/fiber/renderer.js

```diff
@@ -131,6 +133,55 @@ import type {
 import type {Source} from 'react-devtools-shared/src/shared/types';
 import {getStackByFiberInDevAndProd} from './DevToolsFiberComponentStack';
 
+const FIBER_INSTANCE = 0;
+// const VIRTUAL_INSTANCE = 1;
+
+// This type represents a stateful instance of a Client Component i.e. a Fiber pair.
+// These instances also let us track stateful DevTools meta data like id and warnings.
+type FiberInstance = {
+  kind: 0,
+  id: number,
+  parent: null | DevToolsInstance, // virtual parent
+  componentStack: null | string,
```

Comment:
This is only used by `getSourceForFiber` and it really doesn't need the full parent stack to do that so it's a bit overkill. The parent stacks appended to consoles aren't cached and they're really not used once we ship `enableOwnerStacks` anyway.

We could probably drop this field.

Reply:
Yup, we can drop this together with `getSourceForFiber` once we enable `enableOwnerStacks` by default